src/test/java/com/amazonaws/encryptionsdk/kms/AwsKmsMrkAwareMasterKeyProviderTest.java

```diff
@@ -224,23 +225,32 @@ public void no_duplicate_mrks() {
 
     @Test
     public void always_need_a_region() {
+      assertThrows(
+          AwsCryptoException.class,
+          () ->
+              AwsKmsMrkAwareMasterKeyProvider.builder()
+                  .withDefaultRegion(null)
+                  .buildStrict("mrk-edb7fe6942894d32ac46dbb1c922d574"));
       AwsKmsMrkAwareMasterKeyProvider.builder()
-          // If default region is set to `null` or not configured
-          // it will pick one up from the environment
-          .withDefaultRegion(null)
+          .withDefaultRegion("us-east-1")
```

Comment:
I don't understand this test. The name is "always need a region", but a region is passed, so I'm confused.

Reply:
Wait! Is this saying "See, `null` fails and `us-east-1` passes, must be right?

This is putting back the test right?

Reply:
That is correct.
Because this library has a weird feature where `mrk-edb7fe6942894d32ac46dbb1c922d574` is 
considered a valid ARN if and only if a region is given.